client/src/frontend/components/composed/EntityHero/styles.js

```diff
@@ -0,0 +1,140 @@
+import styled from "@emotion/styled";
+import { transientOptions } from "helpers/emotionHelpers";
+import { breakpoints } from "theme/styles/variables/media";
+import { respond, containerPrototype } from "theme/styles/mixins";
+import { standaloneHeaderLayout } from "theme/styles/variables/crossComponent";
+
+const BREAKPOINT = breakpoints[60];
+
+const gridGapSmall = "30px";
+const rowGapMedium = "3.159vw";
+const columnGapMedium = "4.839vw";
+const rowGapLarge = "48px";
+const columnGapLarge = "100px";
+const rightColumnWidth = "28%";
```

Comment:
The width for journals and issues on desktop seems narrower. `200px` versus the `~258px` value this computes to.